test/org/sagebionetworks/bridge/models/schedules/ScheduleContextTest.java

```diff
@@ -81,7 +81,7 @@ public void builderWorks() {
         assertEquals(3, context.getMinimumPerSchedule());
         assertEquals("healthCode", context.getCriteriaContext().getHealthCode());
         assertEquals(TestConstants.USER_DATA_GROUPS, context.getCriteriaContext().getUserDataGroups());
-        assertEquals(now, context.getNow());
+        assertEquals(now, context.getStartsOn());
```

Comment:
Updated to test the withContext() method. (This found an error when copying a context where accountCreatedOn is null).